src/templates/_includes/nav.html

```diff
@@ -0,0 +1,44 @@
+<mkt-nav-root>
```

Comment:
Yeah, there didn't seem to be any real benefit from doing it either way, so I went with what we've been doing in marketplace-elements. We could easily switch `mkt-nav-root` to `ul[is="mkt-nav-root"]` in the CSS if there was a compelling reason to do so.